integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -763,7 +763,7 @@ public static void verifyBeforeDeletion(Domain domain) throws Exception {
     k8sTestUtils.verifyNoReplicaSets(domain1LabelSelector);
     k8sTestUtils.verifyServices(domain1LabelSelector, 5);
     k8sTestUtils.verifyPvcs(domain1LabelSelector, 1);
-    k8sTestUtils.verifyIngresses(domainNs, domainUid, domain1LabelSelector, 1);
+    k8sTestUtils.verifyIngresses(domainNs, domainUid, domain1LabelSelector, 0);
```

Comment:
Can you just remove this line entirely?  And, remove verifyIngresses() from k8sTestUtils...

Reply:
Done.